<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingDialog</class>
 <widget class="QDialog" name="settingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src/icons.qrc">
    <normaloff>:/mainui/icon/MainIcon.ico</normaloff>:/mainui/icon/MainIcon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollArea {
	border:none;
	background:white;
}

QCheckBox {
    padding: 2px;margin: 2px;
}
QCheckBox::indicator {
    width: 15px;
    height: 15px;
}
QCheckBox::indicator:unchecked {
    border-radius:4px;
	border-style:solid;
	border-width:2px;
	border-color:#34495E;
  	background-color:rgb(255,255,255);
}
QCheckBox::indicator:checked {
	border-radius:4px;
	border-style:solid;
	border-width:2px;
	border-color:#0078d7; 
	image: url(:/mainui/icon/selected.png);
}
QCheckBox::indicator:hover{
	border-color:#0078d7; 
}
QCheckBox:hover {
	color:rgb(0, 0, 216);
	padding: 3px;
}

QPushButton {
    border-style:none;
    padding:7px;
    border-radius:15px;
    color:#FFFFFF;
    background:#34495E;
}
QPushButton:hover {
    color:#4E6D8C;
    background:#e5e5e5;
	border:2px solid #34495E;
}
QPushButton:pressed {
    color:#000000;
    background:#B8C6D1;
}

/*表右侧的滑条*/
QScrollBar:vertical {
    background:#484848;
    padding:0px;
    border-radius:6px;
    max-width:12px;
}

/*滑块*/
QScrollBar::handle:vertical {
    background:#CCCCCC;
}
/*
滑块悬浮，按下*/
QScrollBar::handle:hover:vertical,QScrollBar::handle:pressed:vertical {
    background:#A7A7A7;
}
/*
滑块已经划过的区域*/
QScrollBar::sub-page:vertical {
    background:444444;
}

/*
滑块还没有划过的区域*/
QScrollBar::add-page:vertical {
    background:5B5B5B;
}

/*页面下移的按钮*/
QScrollBar::add-line:vertical {
    background:none;
}
/*页面上移的按钮*/
QScrollBar::sub-line:vertical {
    background:none;
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:white;</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>454</width>
        <height>414</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>无动作自动锁定时间(分)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_autoLockTime">
            <property name="font">
             <font>
              <family>Consolas</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QSpinBox {
	border-style:solid;
	border-width:2px;
	border-radius:6px;
	border-color:#34495E;
	padding:1px;
	margin: 1px;
}
QSpinBox:hover{
	color:rgb(0, 0, 216);
	border-color:#0078d7;
}
QSpinBox::up-button {
    padding:1px;
    background-color:transparent;
}
QSpinBox::down-button {
    padding:1px;
    background-color:transparent;
}
QSpinBox::down-button {
    padding:1px;
    background-color:transparent;
}
/* 向上箭头*/
QSpinBox::up-arrow {
	image:url(:/mainui/icon/up1.png);
	height: 15px;
}
QSpinBox::up-arrow:hover {
	
	image: url(:/mainui/icon/up0.png);
	height: 15px;
}
QSpinBox::up-arrow:pressed {
	
	image: url(:/mainui/icon/up0.png);
	height: 13px;
}
/*向下箭头*/
QSpinBox::down-arrow {
	image: url(:/mainui/icon/down1.png);
	height: 15px;
}
QSpinBox::down-arrow:hover {
	image: url(:/mainui/icon/down0.png);
	height: 15px;
}
QSpinBox::down-arrow:pressed {
	image: url(:/mainui/icon/down0.png);
	height: 13px;
}
</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>336</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="4" column="1">
           <widget class="QCheckBox" name="cb_autoBackup">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>自动备份</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="cb_deleteTips">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>删除时确认提示</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="cb_showToolTips">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>显示tooltips</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="cb_pushCtrl">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>按住Ctrl键显示密码</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="cb_showLineIndex">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>显示行号</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="cb_useRegExpSearch">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>使用正则表达式搜索</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="cb_mouseClickSelect">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>鼠标单击选中行</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_save">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="src/icons.qrc"/>
 </resources>
 <connections/>
</ui>
